language/Methods.md

```diff
@@ -105,16 +105,23 @@ DoSomething(7, 8, 9, 10);
 ZScript includes an extra method type for descendents of `Actor` called
 *actions*, which are intended to be run from actor states and give extra
 information to the function. Action functions change the meaning of the `self`
-parameter and pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
-refers to the `State` which this action was called from. Here is a chart for
-the meanings of the `self` and `invoker` variables under each scope:
-
-| Scope     | `self` meaning                                                  | `invoker` meaning |
-| -----     | --------------                                                  | ----------------- |
-| None      | The actor this function operates on, ambiguous in some contexts | State owner       |
-| `actor`   | The actor                                                       | The actor         |
-| `item`    | Item owner                                                      | Item itself       |
-| `overlay` | Weapon owner                                                    | Weapon itself     |
-| `weapon`  | Weapon owner                                                    | Weapon itself     |
+parameter and may pass in `invoker` and `stateinfo` parameters as well. `stateinfo`
+refers to the `State` which this action was called from.
+
+Action scopes are a flag field and as such an action function can have multiple scopes.
```

Comment:
This is a little too specific, preferably this would be written as user-facing documentation rather than from an implementation perspective. I would suggest the phrasing "Action functions may have one scope or multiple scopes."